### PR TITLE
Trying to Add Context for Potential Contributors

### PR DESCRIPTION
**Pull request by ryanleesipes**
First pass, feedback welcome.

Added general information on what open source software is and noting that Thunderbird is created by an online community.

### CODE REVIEW

**Comment by ryanleesipes** [comment]
Screenshot:

Screenshot from 2019-06-25 12-47-57

File: website/get-involved/index.html [hunk 1]
```diff
@@ -34,6 +35,25 @@ <h2>{{ self.page_desc() }}</h2>
...
+                <p>{{ _('Thunderbird is created by an online community of volunteers.') }}</p>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col full">
+                <section>
+                    <h3>{{ _('What is Open Source Software?') }}</h3>
+                    <p>{{ _('Thunderbird is open source software, meaning that anyone can inspect, modify, enhance it, and share it. But the fun is not just for programmers!
+                        Thunderbird is created by a community of designers, translators, people who do documentation and support, as well as developers. We invite anyone to come
+                        and participate in the creation of Thunderbird regardless of your skillset, we’re sure there is something you can add to our community
```
Comment (Sancus) [inline website/get-involved/index.html]: This is a bit of a run-on, I think. "We invite anyone to come and participate in the creation of Thunderbird. Regardless of your skillset, we're sure there is something you can add to our community and project." sounds better to me.

Comment (ryanleesipes) [reply website/get-involved/index.html]: I agree. Will change.

File: website/get-involved/index.html [hunk 2]
```diff
@@ -34,6 +35,25 @@ <h2>{{ self.page_desc() }}</h2>
...
+            <div class="col full">
+                <p>{{ _('Thunderbird is created by an online community of volunteers.') }}</p>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col full">
+                <section>
+                    <h3>{{ _('What is Open Source Software?') }}</h3>
+                    <p>{{ _('Thunderbird is open source software, meaning that anyone can inspect, modify, enhance it, and share it. But the fun is not just for programmers!
+                        Thunderbird is created by a community of designers, translators, people who do documentation and support, as well as developers. We invite anyone to come
```
Comment (Sancus) [inline website/get-involved/index.html]: "people who do documentation" seems awkward. In theory they're called technical writers, but that's a little bit off putting to someone who isn't a professional, so maybe:

"Thunderbird is created by a community of designers, translators, documentation writers and support people in addition to developers."

Comment (ryanleesipes) [reply website/get-involved/index.html]: Good feedback, agreed, resolved.

**Pull request merged**